### DISCUSSION TITLE
Welcome to mmrm Discussions!

### DISCUSSION THREAD

**Discussion by danielinteractive** [openpharma/mmrm#314] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by henrikibster** [comment]
Hi all. 

I have started using the mmrm package in connection with some simulation studies I am performing. I am fitting a model along the lines of : 
mmrm.fit <- mmrm(
  formula = CFB ~ treat*week.factor + obs.baseline*week.factor + us(week.factor | subj.id),
  data = df[planned.week != 0] 
)

Based on this model I will derive a complex quantity, and would like to employ bootstrapping techniques to get at confidence interval. I have been playing around with the lmeresampler pacakge, however that only works on lme4 and nlme objects. 

Have you considered to implement methods for bootstrapping, og teaming up with the lmeresampler-folks making their package work on mmrm objects?

Kr, Henrik

**Reply by danielinteractive** [reply]
Thanks @henrikibster for the question and idea!
Honestly that never came up so far.
I would guess if you give us a few details of what would be needed as interface to work with that bootstrap package we could look into it?
Also I guess a manual bootstrap implementation might not be too complex here.

**Comment by henrikibster** [comment]
I have actually already done a manual bootstrap implementation for my application. 
The lmeresample package however has several different bootstrap methods implemented, so I thougth it would be a nice feature if the two packages could 'talk together'. 
I have not looked into the actual implementation in the lmeresample package, and my R skills are so limited that it would probably be a futile quest for me to start doing that.

**Reply by danielinteractive** [reply]
Thanks @henrikibster - understood. If you have any example code for your manual path that you could share it would also be great, and maybe also the nlme based lmeresample package usage.
Moving this to a new issue: https://github.com/openpharma/mmrm/issues/372 so we can continue discussion there.
